src/cli/cli.ts

```diff
@@ -81,22 +143,6 @@ for (const {alias, name, type} of browsers) {
       });
 }
 
-program
-    .command('codegen [url]')
-    .description('open page and generate code for user actions')
-    .option('-o, --output <file name>', 'saves the generated script to a file')
-    .option('--target <language>', `language to use, one of javascript, python, python-async, csharp`, language())
-    .action(function(url, command) {
-      codegen(command.parent, url, command.target, command.output);
-    }).on('--help', function() {
-      console.log('');
-      console.log('Examples:');
-      console.log('');
-      console.log('  $ codegen');
-      console.log('  $ codegen --target=python');
-      console.log('  $ -b webkit codegen https://example.com');
-    });
-
 program
     .command('screenshot <url> <filename>')
```

Comment:
Don't we need all other options for screenshot as well? And for pdf? And cr/ff/wk aliases?